File: go/linear-search/LinearSearchList.go
package main

import "fmt"

func linearSearch(list []int, number int) bool {
	for i := 0; i < len(list); i++ {
		if list[i] == number {
			return true
		}
	}
	return false
}

func main() {
	number := 20
	list := []int{100, 20, 30, 23, 434, 564, 65, 54, 52, 40, 9, 3, 50, 2}
	fmt.Print(linearSearch(list, number))
}

File: js/linear-search/LinearSearchList.js
function linear_search(lista, numero){
    for (let i = 0; i < lista.length; i++){
        if (lista[i] === numero){
            return true;
        }
    }
    return false;
}


const numbers = [1, 2, 3, 4, 5, 65, 76, 87, 90, 230, 304, 340, 540, 4345];

const numero = 85;

console.log(linear_search(numbers, numero));
